packages/mdc-menu-surface/adapter.ts

```diff
@@ -21,7 +21,7 @@
  * THE SOFTWARE.
  */
 
-/* eslint no-unused-vars: [2, {"args": "none"}] */
+import {MenuDimensions, MenuDistance, MenuPoint} from './constants';
 
 /**
  * Adapter for MDCMenuSurface. Provides an interface for managing
```

Comment:
remove 
```
* Additionally, provides type information for the adapter to the Closure
 * compiler.
 *	
```

We've also been removing the list of 

```
* - classes
 * - dom
 * - focus
 * - position
 * - dimensions
 * - event handlers
```

but I dont think its necessary

Reply:
Done